pyveda/training.py

```diff
@@ -91,6 +91,14 @@ def id(self):
     def label(self):
         return self.data['label']
 
+    @property
+    def bounds(self):
+        return self.data['bounds']
+
+    @property
+    def tile_coords(self):
+        return self.data['tile_coords']
```

Comment:
So in some cases tile_coords wont be in data. Can you change this to something a bit safer? 

eg: self.data.get('tile_coords')

or you may need to check like:

```
if 'tile_coords' in self.data:
    return self.data['tile_coords']
else:
    return None
```

Reply:
makes sense thank you! I've added the check for tile coords

Reply:
What about bounds for ungeoreferenced imagery? We haven't tackled that case yet but might want to be safe about that one too.

Reply:
yes both